frontstage/templates/surveys/surveys-confirm-organisation.html

```diff
@@ -10,18 +10,23 @@ <h1 id="confirm-org-title" class="ons-u-fs-xl">Confirm organisation and survey d
 
     {% include('partials/business_survey_credentials.html') %}
 
-    {{
-        onsButton({
-            "url": url_for('surveys_bp.add_survey_submit', encrypted_enrolment_code=context.encrypted_enrolment_code) ,
-            "text": "Confirm",
-            "type": "submit",
-            "classes": "ons-u-mt-l ons-u-mb-s",
-            "id": "confirm_survey_button",
-            "submitType": "timer"
-        })
-    }}
-
-    <div><a id="cancel_button" href="{{ url_for('surveys_bp.get_survey_list', tag='todo') }}">Cancel</a></div>
+    <div class="ons-grid ons-grid-flex ons-grid--gutterless ons-grid-flex--vertical-center ons-grid--no-wrap@s ons-u-mt-m ons-u-mb-m">
+        <div class="ons-grid__col">
+            {{
+                onsButton({
```

Comment:
Again, I can't seem to pick up the change in the onsButton

Reply:
As abox

Reply:
As above